server/routers/routes/sampleRoute.ts

```diff
@@ -0,0 +1,4 @@
+export default function(req, res) {
```

Comment:
this uses a regular function def while the other uses arrow functions - should we use the same for consistency?

Reply:
nice catch  👍 , @matjack1 can you please make changes?